core/indexer/elasticsearchDatabase.go

```diff
@@ -278,14 +278,17 @@ func (esd *elasticSearchDatabase) SaveMiniblocks(header data.HeaderHandler, body
 	}
 
 	buffInsert, buffUpdate := serializeBulkMiniBlocks(header.GetShardID(), miniblocks)
-	err := esd.dbWriter.DoBulkRequest(&buffInsert, miniblocksIndex)
-	if err != nil {
-		log.Warn("indexing bulk of miniblock insert ", "error", err.Error())
+	if buffInsert.Len() != 0 {
```

Comment:
In the end we are going to leave 2 separate requests for insert and update? I remember that in the end we realised that it was good in one request also. But not an issue if we do it separately.

Reply:
You right is good to do only one request when we index miniblocks. Fixed .